library/views.py

```diff
@@ -1,3 +1,28 @@
-from django.shortcuts import render
+import json, requests
+from django.http    import JsonResponse
 
-# Create your views here.
+from library.models import Library, LibraryBook
+from book.models    import Book
+from django.views   import View
+from share.decorators import check_auth_decorator
+
+class MyLibraryView(View):
+    @check_auth_decorator
+    def post(self,request):
+        data = json.loads(request.body)
+        try :
+            user      = request.user
+            book_id   = data['book_id']
+            library   = Library.objects.filter(user_id=user)
+
+            if not library:
+                return JsonResponse({'message':'NOT_EXIST_LIBRARY'}, status=400)
+            if LibraryBook.objects.filter(book_id=book_id, library_id=library.first().id).exists():
+                return JsonResponse({'message':'ALREADY_BOOK'}, status=400)
+            book_save  = LibraryBook.objects.create(
+                book_id    = book_id,
+                library_id = library.first().id
+                    )
```

Comment:
```suggestion
            book_save  = LibraryBook.objects.create(
                book_id    = book_id,
                library_id = library.first().id
            )
```
괄호가 한 인덴트 더 밖에 있는 것이 가독성이 높아보입니다 :) 안에 코드가 괄호에 포함된다는 의미를 주기 위해서요 !

Reply:
넵 수정하였습니다!